gtda/externals/python/ripser_interface.py

```diff
@@ -204,26 +197,20 @@ def ripser(X, maxdim=1, thresh=np.inf, coeff=2, metric="euclidean",
 
     """
     if n_perm and sparse.issparse(X):
-        raise Exception(
-            "Greedy permutation is not supported for sparse distance matrices"
-        )
+        raise Exception("Greedy permutation is not supported for sparse "
+                        "distance matrices")
     if n_perm and n_perm > X.shape[0]:
-        raise Exception(
-            "Number of points in greedy permutation is greater"
-            " than number of points in the point cloud"
-        )
+        raise Exception("Number of points in greedy permutation is greater "
+                        "than number of points in the point cloud")
     if n_perm and n_perm < 0:
-        raise Exception(
-            "Should be a strictly positive number of points in the greedy "
-            "permutation"
-        )
+        raise Exception("There should be a strictly positive number of points "
+                        "in the greedy permutation")
 
     idx_perm = np.arange(X.shape[0])
     r_cover = 0.0
     if n_perm:
-        idx_perm, lambdas, dperm2all = get_greedy_perm(
-            X, n_perm=n_perm, metric=metric
-        )
+        idx_perm, lambdas, dperm2all = \
+            get_greedy_perm(X, n_perm=n_perm, metric=metric)
```

Comment:
The indentation is changed because it was not to the library rules ?

Reply:
Well spotted.  We are not that restrictive about indentation style -- the only criterion is that running `flake8` to check linting should not give errors.  My personal approach is that the person or people who are most responsible for part the code get the freedom to use whatever indentation they prefer.

Reply:
I have no problem reverting these if you find it *less* readable now.

Reply:
I'm completely fine with the changes, if they reflect the indentation of the rest of the library. 
I'm not sure that this file was indented as the rest of the library.

For me it's perfectly readable :+1:

Reply:
@MonkeyBreaker the library does not really have uniform indentation styles because we don't enforce one. Do you think it's very important?

Reply:
Well if we don't have one, then the changes are perfectly fine.
But maybe in an issue we should discuss this between maintainers, but I think that if the `flake8` check pass already, it's good !